pkg/utils/utils.go

```diff
@@ -4,6 +4,7 @@ import (
 	"bytes"
 	"encoding/json"
 	"fmt"
+	sriovtypes "github.com/k8snetworkplumbingwg/sriov-cni/pkg/types"
```

Comment:
nit: move it outside of this group

Reply:
yep for some reason my IDE stop creating the nice groups...

done :)